src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if ((Objects.equals(email, "bob@i.ua")) || (Objects.equals(email, "alice@i.ua"))) {
+            return Objects.equals(password, "1234");
+        }
+
```

Comment:
I would say here we need to build our `login` implementation on top of the `UserService.findByEmail` method.